examples/hello-compute/main.rs

```diff
@@ -1,19 +1,8 @@
-use std::{convert::TryInto as _, str::FromStr};
 use zerocopy::AsBytes as _;
 
 async fn run() {
-    env_logger::init();
-
-    // For now this just panics if you didn't pass numbers. Could add proper error handling.
-    if std::env::args().len() == 1 {
-        panic!("You must pass a list of positive integers!")
-    }
-    let numbers: Vec<u32> = std::env::args()
```

Comment:
we could move that into `main` and have `run` receiving this slice in an argument?

Reply:
Yeah we could easily add this back in. I just removed it for now because we need to pick some values at compile-time for wasm anyway